package.json

```diff
@@ -5,7 +5,7 @@
     "email": "azsdkteam@microsoft.com",
     "url": "https://github.com/Azure/ms-rest-azure-js"
   },
-  "version": "0.11.0",
+  "version": "0.11.101",
```

Comment:
intended?

Reply:
Nope. Fixed in https://github.com/Azure/ms-rest-azure-js/pull/41/commits/7c96fe21b1c324d35bd0c87689279b4f363f709e.